pkg/operator/sync.go

```diff
@@ -286,6 +294,29 @@ func newPodTemplateSpec(config *OperatorConfig, features map[string]bool) *corev
 		},
 	}
 
+	volumes := []corev1.Volume{
+		{
+			Name: kubeRBACConfigName,
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "kube-rbac-proxy",
+					},
+					DefaultMode: pointer.Int32Ptr(420),
```

Comment:
Has this DefaultMode been inspired by somewhere else?

Reply:
https://github.com/openshift/machine-api-operator/blob/08a4e21a14aaf27f6ead3fdc8f6fff5e0000d5eb/install/0000_30_machine-api-operator_11_deployment.yaml#L88-L99 State of all those mounts are managed by kubernetes resources, so this is `readOnly`